src/overlays/actors/ovl_En_Tg/z_en_tg.c

```diff
@@ -76,37 +91,191 @@ static DamageTable D_809901F8 = {
     /* Powder Keg     */ DMG_ENTRY(0, 0x0),
 };
 
-#endif
+static AnimationInfoS sAnimationInfo = { &gHoneyAndDarlingIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 };
```

Comment:
`EnTg_ChangeAnim` seems to be treating the animationInfo parameter as an array, so it would be better to change this to be an array of only one element
```suggestion
static AnimationInfoS sAnimationInfo[] = {
    { &gHoneyAndDarlingIdleAnim, 1.0f, 0, -1, ANIMMODE_LOOP, 0 },
};
```